### DISCUSSION TITLE
Is it possible to turn my favorite program into the best?!!!!

### DISCUSSION THREAD

**Discussion by Joh45456464** [freeplane/freeplane#715] [Help]
**Open source and free software should be better than paid simply because it is for the most numerous and poorest segment.**

1. can insert  folating node by double click any where or (can custom chose ctrl+double click ---- or double click only in preferences).

2. Mouse selection:
The ability to select any group of nodes by using the mouse to draw a rectangle on it

3. add ctrl+shift  for **right to left **languages**** .

4.  Pleaseeee add other plans  like tree map fishbone org chart timeline any .. Also, merging all schematics
All this is very important for some people because it is useful for those who study
Therefore, if you please, I hope that you will add that, and if you do not, at least guide us to do so (apart from programming, I do not know much about it)  .

5. convert to  freeboard like scapple  / ayoa.

So thanks for your software .

**Comment by euu2021** [comment]
>convert to freeboard like scapple / ayoa

It's possible to have free positioned nodes. What exactly do you have in mind?

**Comment by Joh45456464** [comment]
You can understand me by looking at these programs.

https://www.youtube.com/watch?v=Iy49L8a8i7M&t=39s&ab_channel=Ayoa

https://www.youtube.com/watch?v=zvP6c7AFx_c&ab_channel=scrivenerapp

sticky note - audio note  
and you can see several divisions in one map

![KHDHNDbm65](https://user-images.githubusercontent.com/108882090/194170773-0ab4efc9-0ca5-4204-9e47-d5000d13a11c.png)


All I wish is to see freeplane the best program.

Thanks.

**Reply by euu2021** [reply]
About the different layouts, it may happen in the future. Take a look at #687 

About using free positioned nodes to create a more Scapple like experience, take a look at [this thread](https://sourceforge.net/p/freeplane/discussion/758437/thread/5c60cc647a), where I showed an example. It's already possible to do interesting things using the free positioned nodes feature.
